### PR TITLE
index.html: move logo

### PR DESCRIPTION
**Pull request by danes44**
This modify move the coala logo
Closes https://github.com/coala/community/42

### CODE REVIEW

File: templates/index.html [hunk 1]
```diff
@@ -1,5 +1,7 @@
 <title>Community website</title>
+
```
Comment (andrewda) [inline templates/index.html]: Unneccessary change

File: templates/index.html [hunk 2]
```diff
@@ -1,5 +1,7 @@
...
 <body>
+<img src="static/org_logo.png" alt="coala">
```
Comment (andrewda) [inline templates/index.html]: Can't have the word "coala". Use something like "org logo" for the alt instead.

**Comment by andrewda** [comment]
Also fix the closes link in your commit. `https://github.com/coala/community/42` is a 404, should be `https://github.com/coala/community/issues/42`

File: templates/index.html (hunk 1 above)
Comment (danes44) [reply templates/index.html]: Thanks for review my task :)

**Comment by jayvdb** [comment]
In your commit message, please add a blank line above "`Closes https://github.com/coala/community/issues/42`"

**Comment by jayvdb** [comment]
Here is the preview:  https://deploy-preview-68--coala-community.netlify.com/ 

Could you reduce the height and width of the logo in the HTML/CSS.  Leave the image file with its current dimensions.

Optional, you can also remove the logo from `gci/` page.

**Pull request merged**